spec/features/shelterreviews/shelter_review_index_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe "when I visit a shelter show page", type: :feature do
+  it "I see a list of reviews for that shelter" do
+
+    shelter1 = Shelter.create(name: "Dog-Haven",
+                               address:  "1234 Barkers Way",
+                               city: "Beagle",
+                               state: "MA",
+                               zip: "01001")
+
+    shelter2 = Shelter.create(name: "The Cat Palace",
+                               address: "998 Kings Ct",
+                               city: "Kings Landing",
+                               state: "CA",
+                               zip: "90210")
+
+    review1 = shelter1.reviews.create!(review_image: "https://via.placeholder.com/150",
```

Comment:
Also to make a note of: Tony wanted us to make sure this was formatted properly.  Right now it isn't looking the way he was talking about...